### PR TITLE
[MRG] fix one deprecation warning

### PR DESCRIPTION
**Pull request by jeromedockes**
see https://github.com/nilearn/nilearn/pull/1766#issuecomment-431546544

### CODE REVIEW

**Comment by bthirion** [comment]
LGTM.

File: nilearn/plotting/js_plotting_utils.py [hunk 1]
```diff
@@ -5,12 +5,15 @@
 
 import os
 import base64
-import cgi
 import webbrowser
 import tempfile
 import warnings
 import subprocess
 import weakref
+try:
```
Comment (GaelVaroquaux) [inline nilearn/plotting/js_plotting_utils.py]: Maybe a comment to say why this try except? It will make it easier to clean this in the future.

Comment (kchawla-pi) [reply nilearn/plotting/js_plotting_utils.py]: Done.

**Comment by GaelVaroquaux** [comment]
Great. Thank you to both of you. The CI is green. I am merging!

**Pull request merged**